.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- According to the project requirements, your `gitignore` file should ignore a `test.md` file, please add the `test.md` file and ensure it is ignored by Git. Please fix this 🙏

Reply:
Done👍😊